packages/browser/src/Component/Itowns/3DTiles/3DTilesUtils.js

```diff
@@ -1,5 +1,133 @@
 import * as THREE from 'three';
 
+/**
+ * Create a geometric outliers for each tile of a tileset when its geometry is loaded
+ * Each outlier tile geometry does not keep the batchid for each outliers.
+ * May be slow to create / load.
+ *
+ * @param {THREE.Object3D} tile  A 3DObject (should be a tile send by Tilesmanager.EVENT_TILE_LOADED)
+ * @param {number} threshOldAngle  An edge is only rendered if the angle (in degrees) between the face normals of the adjoining faces exceeds this value. default = 1 degree.
+ */
+export function appendWireframeToTileset(tile, threshOldAngle = 30) {
+  if (
```

Comment:
remove these if and remove the for loop to write something like:

tile.traverse((child)=>{
if(child.geometry && child.geometry.isBufferGeometry){
//do stuff
})

Reply:
done